demo/python/multiclient-cluster-remote-control/run-distributed.sh

```diff
@@ -1 +1 @@
-../../../host/dmlc-core/tracker/dmlc-submit --log-level DEBUG --cluster ssh --host-file hosts.config --num-workers $1 --worker-memory 4g python3 mc-distr.py
+../../../host/dmlc-core/tracker/dmlc-submit --log-level DEBUG --cluster ssh --host-file hosts.config --num-workers $1 --worker-memory 4g python3 server/enclave_serve.py
```

Comment:
This script starts the tracker on the orchestrator machine and sets up the cluster. A different script starts the orchestrator. We can combine them but I left them separate to make it easier to understand what's going on / for easier debugging.

Reply:
Perhaps combine the scripts and add comments within the script? Less clutter IMO.

Reply:
Oh actually I take that back -- I don't think we can combine them since the command to launch the tracker will block until the demo has finished, and we need the orchestrator to run the demo. 

So all this script will do is start the tracker, and the user can start the orchestrator by running `python3 start_orchestrator.py`. I'm writing up some documentation for the demo and will add it in a later PR.

Reply:
Cool.